frontend/src/store/course/actions.ts

```diff
@@ -272,17 +286,26 @@ const updateIsMentorChoosingEnabled = createAsyncThunk<
   async (id, { extra, getState }) => {
     const {
       auth: { user },
-      course: { mentors },
+      course: { mentors, course, activeInterviewsCategoryIds },
     } = getState();
 
     const isMentor = mentors.some(
       (mentor) => mentor.id === (user as UserWithPermissions).id,
     );
+    const isInterviewProcessActive = activeInterviewsCategoryIds.some(
+      (categoryId) =>
+        categoryId === (course as CourseGetResponseDto).courseCategoryId,
+    );
```

Comment:
To avoid ugly line wraps
```suggestion
    const isInterviewProcessActive = activeInterviewsCategoryIds.some((categoryId) => {
      return categoryId === (course as CourseGetResponseDto).courseCategoryId;
    });
```

Reply:
@drKravets please fix this comment

Reply:
@Liza-Veis  I've tried to change it to formatting Andrii suggested but it was auto-formatted into 2 lines while commit. Not sure how to avoid this.